tests/src/whisk/core/limits/MaxActionDurationTests.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2015-2016 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package whisk.core.limits
+
+import scala.concurrent.duration.DurationInt
+
+import org.junit.runner.RunWith
+import org.scalatest.junit.JUnitRunner
+
+import common.TestHelpers
+import common.TestUtils
+import common.Wsk
+import common.WskProps
+import common.WskTestHelpers
+import whisk.core.entity._
+import spray.json.DefaultJsonProtocol._
+import spray.json._
+import whisk.http.Messages
+import whisk.core.entity.TimeLimit
+
+/**
+ * Tests for action duration limits. These tests require a deployed backend.
+ */
+@RunWith(classOf[JUnitRunner])
+class MaxActionDurationTests
+    extends TestHelpers
+    with WskTestHelpers {
+
+    implicit val wskprops = WskProps()
+    val wsk = new Wsk
+
+    Map("node" -> "helloDeadline.js", "python" -> "timedout.py").foreach {
```

Comment:
as noted earlier, these tests are sufficient.
if you add a jar for a java version, you can add it to the map to complete the runtime coverage.